src/OnlineSales/appsettings.json

```diff
@@ -1,7 +1,7 @@
 ﻿{
   "AllowedHosts": "*",
   "MigrateOnStart": true,
-  "Plugins": [ "OnlineSales.Plugin.ReverseProxy", "OnlineSales.Plugin.EmailSync" ],
+  "Plugins": [ "OnlineSales.Plugin.ReverseProxy", "OnlineSales.Plugin.EmailSync", "OnlineSales.Plugin.XLTools" ],
```

Comment:
@Mixer57 external plugins should never be included into the core configuration by default!